lib/tasks/plugins.rake

```diff
@@ -25,6 +25,28 @@ namespace :plugin do
       ActiveRecord::Tasks::DatabaseTasks.migrations_paths << path
     end
   end
+
+  task :rubocop, [:engine, :junit] => :environment do |t, args|
+    unless args[:engine]
+      abort("You must specify the name of the plugin (e.g. rake plugin:rubocop['my_plugin'])")
+    end
+
+    engine = "#{args[:engine].tr('-', '_').camelize}::Engine".constantize
```

Comment:
This is a little hacky, see https://github.com/theforeman/foreman/pull/6802/files#diff-70412c0ba2a8f8460972ba17c0fa4828R60-R68 for a better implementation.

Reply:
Copied this from https://github.com/theforeman/foreman/blob/6bd234dbebfe728507078afb9561f4e691f51989/lib/tasks/plugin_apipie.rake#L8-L9 so it looks like we would benefit from a `Foreman::Plugin.by_name(name)` method.